configure.ac

```diff
@@ -1491,10 +1514,11 @@ AC_CHECK_HEADER([sys/mman.h],
 AC_CHECK_FUNC([pwrite], [AC_DEFINE([HAS_PWRITE])])
 
 ## -fdebug-prefix-map support by the C compiler
-AS_CASE([$CC,$host],
+AS_CASE([$ocaml_cv_cc_vendor,$host],
```

Comment:
Review note: this change affects all architectures, but actually it only matters when the cc_vendor is xlc and sunc, so it is actually rather safe.

Reply:
`xlc` and `sunc` are the oldest compilers of all 5 supported.